Frontend/CO.CDP.OrganisationApp/Authorization/CustomScopeHandler.cs

```diff
@@ -18,12 +19,11 @@ protected override async Task HandleRequirementAsync(AuthorizationHandlerContext
                 // The UserInfoService is scoped, but authorization is a singleton, so we need to work around that with a ServiceScopeFactory
                 using (var serviceScope = serviceScopeFactory.CreateScope())
                 {
-                    IUserInfoService _userInfo = serviceScope.ServiceProvider.GetRequiredService<IUserInfoService>();
+                    var tenantClient = serviceScope.ServiceProvider.GetRequiredService<ITenantClient>();
 
-                    var userScopes = await _userInfo.GetUserScopes();
-
-                    // GetOrganisationUserScopes below will 404 for support admins who do not have an org of their own
-                    // Therefore we deal with support admin permissions first because we can exit early
+                    var tenantLookup = await tenantClient.LookupTenantAsync();
```

Comment:
@jakzal the old implementation inside `GetOrganisationUserScopes` used to cache the tenant lookup for the lifetime of the current http request in memory. This is no longer in place, so we will have some pages such as supplier information where we will end up calling this authorization handler and therefore tenant lookup api 9 times (8 calls to the `<authorize>` tag helper and one `[Authorize]` attribute on the razor page.

This seems excessive to me. Even if we are content to not deal with it in this pull request - what is our strategy for this going forward? Or is this a complete non issue and we're happy to make many calls to APIs like this?

Reply:
Will mark this as resovled for a second as it's more of a question, not necessarily blocking this PR